oak-jcr/src/test/java/org/apache/jackrabbit/oak/jcr/NamePathTest.java

```diff
@@ -212,6 +218,47 @@ public void testSpaceInNames() throws RepositoryException {
         session.getNode(n.getPath());
     }
 
+    @Test
+    @Ignore("OAK-10544")
```

Comment:
Shouldn't we rather add it to the https://svn.apache.org/repos/asf/jackrabbit/trunk/jackrabbit-jcr-tests/? It doesn't use any Oak API IIUC and JR2 should work just fine with that.

Reply:
That's a good point in the mid term: those reside in Jackrabbit though, so it would take some time to have a release that contains them and can be used in Oak. But yes.

Also, in the meantime I learned that there maybe a more specific test class I could use; I'll update the PT soonish.